e2e/testvalues/values.go

```diff
@@ -7,8 +7,9 @@ import (
 )
 
 const (
-	StartingTokenAmount int64 = 10_000_000
-	IBCTransferAmount   int64 = 10_000
+	StartingTokenAmount int64  = 10_000_000
+	IBCTransferAmount   int64  = 10_000
+	ImmediatelyTimeout  uint64 = 1
```

Comment:
not sure if this makes sense in the const file as we wouldnt necessarily change this value (changing the value would change the function) -- maybe you could just add a comment in the code above the line this value is used to explain that it means immediate timeout?